extensions/wikia/Lightbox/scripts/Lightbox.js

```diff
@@ -76,13 +76,6 @@
 			// cache re-used DOM elements and templates for this modal instance
 			Lightbox.cacheDOM();
 
-			// Init ads in Lightbox
-			if ($('#MODAL_RECTANGLE').length && Lightbox.ads.showAds()) {
-				Lightbox.openModal.lightbox.addClass('show-ads');
```

Comment:
From what I can see from code -- it was used only by `MODAL_RECTANGLE` but, definitely, it's worth to do happy-path testing (if we'd go with removing those styles).